lib/idl/idl_checkers.es6.js

```diff
@@ -307,6 +306,10 @@ Right: ${right.url}`);
       }
       if (olas.length > 0) {
         logger.error(`${name}.${lm.name}`);
+        logger.info(`Left has:
```

Comment:
Are deliberately putting part of the message at the error level and part at the info level? If so, that's fine so long as a description of what the error is appears in the error line above.